From b14cd84e96fff1083837a70548ff9e18472c938b Mon Sep 17 00:00:00 2001
From: Zane Bitter <zane.bitter@gmail.com>
Date: Wed, 9 Mar 2011 23:55:06 -0500
Subject: [PATCH] Fix minor errors in man page documentation for corosync.conf

* Correct 'See Also' reference to corosync.conf(5) in corosync(8) man page
* Update path to default config (now /etc/corosync/corosync.conf)

Signed-off-by: Zane Bitter <zane.bitter@gmail.com>
Reviewed-by: Steven Dake <sdake@redhat.com>
(cherry picked from commit 230352512595462ea01dfa27bc8c7e81e5adbff3)
---
 man/corosync.8      |    2 +-
 man/corosync.conf.5 |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/man/corosync.8 b/man/corosync.8
index a5b1fcb..c45cc56 100644
--- a/man/corosync.8
+++ b/man/corosync.8
@@ -51,7 +51,7 @@ Do not set process priority.
 Display version and SVN revision of Corosync and exit.
 .SH SEE ALSO
 .BR corosync_overview (8),
-.BR corosync.config (5),
+.BR corosync.conf (5),
 .BR cpg_overview (8),
 .BR votequorum_overview (8),
 .BR sam_overview (8),
diff --git a/man/corosync.conf.5 b/man/corosync.conf.5
index bad9700..b6fd0a0 100644
--- a/man/corosync.conf.5
+++ b/man/corosync.conf.5
@@ -37,7 +37,7 @@
 corosync.conf - corosync executive configuration file
 
 .SH SYNOPSIS
-/etc/corosync.conf
+/etc/corosync/corosync.conf
 
 .SH DESCRIPTION
 The corosync.conf instructs the corosync executive about various parameters
@@ -638,7 +638,7 @@ required.
 
 .SH "FILES"
 .TP
-/etc/corosync.conf
+/etc/corosync/corosync.conf
 The corosync executive configuration file.
 
 .SH "SEE ALSO"
-- 
1.7.1

